docs/source/api_reference/dists_kernels.rst

```diff
@@ -181,7 +181,7 @@ See docstring of ``AggrDist`` and ``FlatDist``.
     AggrDist
     FlatDist
 
-Advanced time series kernels that cannot be expressed as aggrgates or flat applicates:
```

Comment:
not a typo

Reply:
`aggrgates` is

Reply:
no, applicates

Reply:
I mean, it is correcting applicates to applicants which is a false positive.

It is not correcting aggrgates to aggregates, which a false negative.

Reply:
I mean on the same line "aggrgates" is a typo, "aggregates" is the correct spelling

Reply:
yes, that's what I try to say in my last comment.

"applicates" -> "applicants", should not be corrected. "applicates" was correct.
So this is a *false positive*, because the spellchecker detects it but should not.
In my initial comment I was referring to this one.

"aggrgates" -> "aggregates" should be corrected, "aggrgates" is incorrect.
So this is a *false negative*, because the spellchecker does not detect it but should.

In my initial comment I overlooked this one, thank you for pointing this out, @szepeviktor!

Having said that, this is a very very specialized PR and a very peculiar location to help out at, but thanks nonetheless. Might I convince you to contribute to `sktime` in the future?

Reply:
> Might I convince you to contribute to `sktime` in the future?

I am highly blind to business features - what is sadly the most important for you.
All software is the same cake to me, they just have different (thin) icing on top.

What I can do is 
- care about every byte in the repo
- listen to what the (Python) ecosystem says
- run never seen before tests
- find design errors
- find implementation errors
- invent end-to-end tests
All [my thoughts](https://github.com/stars/szepeviktor/lists/theory) and tools are public here in GitHub.

The submerged part of your ice berg is my territory.

Reply:
🥊 cakes vs. ice bergs

Reply:
all of these sound very nice - everyone wants to eat the cake but not enough iceberg miners...

the largest impact is perhaps bugfixes?
https://github.com/sktime/sktime/projects/18

There are also some larger projects and design questions if interested - we can chat at one of the meetups.